plugins/by-name/neoconf/default.nix

```diff
@@ -0,0 +1,92 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "neoconf";
+  originalName = "neoconf.nvim";
+  package = "neoconf-nvim";
+
+  maintainers = [ lib.maintainers.BoneyPatel ];
```

Comment:
> `error: attribute 'BoneyPatel' missing`

Guessing you plan on rebasing once one of your other PRs is merged?

Reply:
Otherwise you could rebase on e84a38aca01366c6d6a9c188f9e05e9d40ebc33f or 4f9a18e28c25727782104021b1f7835c7659f0a4

Reply:
Rebased, good call.